BuildAndTest.proj

```diff
@@ -33,7 +33,7 @@
     <MSBuild BuildInParallel="true"
              Condition="'$(ManualTest)' == ''"
              Projects="$(SamplesSolution)"
-             Properties="$(MSBuildCommonProperties)"
+             Properties="$(MSBuildCommonProperties);DeployVsix=false"
```

Comment:
As pointed out in the mail thread, this is the wrong variable name. Setting this doesn't do anything.